includes_custom_resources/includes_custom_resources_syntax.rst

```diff
@@ -5,25 +5,20 @@
 A custom resource is defined as a |ruby| file and is located in a cookbook's ``/resources`` directory. This file
 
 * Declares the properties of the custom resource
-* Loads current properties, if the resource already exists
 * Defines each action the custom resource may take
 
-The syntax for a custom resource is. For example:
+The syntax for a basic custom resource utilizing built-in chef resources is. For example:
 
 .. code-block:: ruby
 
    property :name, RubyType, default: 'value'
 
-   load_current_value do
-     # some Ruby
-   end
-
    action :name do
-    # a mix of built-in Chef resources and Ruby
+    # built-in Chef resources
    end
 
    action :name do
-    # a mix of built-in Chef resources and Ruby
+    # built-in Chef resources
    end
 
-where the first action listed is the default action.
\ No newline at end of file
+where the first action listed is the default action.
```

Comment:
I kind of think we should just teach people to be explicit about the default action.